components/layout/style/index.less

```diff
@@ -114,6 +114,7 @@
 
         &-right {
           left: -@layout-zero-trigger-width;
+          border-radius: @border-radius-base 0 0 @border-radius-base;
```

Comment:
no rotate then?

Reply:
use one of them can work well as expected
difference is `rotate` will make menu icon inverted, but `border-radius` will not

Reply:
https://codesandbox.io/s/antd-reproduction-template-d34hf

Reply:
👌